node/bin/src/config.rs

```diff
@@ -62,6 +62,14 @@ pub struct GeneralConfig {
     #[config(default_t = 3312)]
     pub prometheus_port: u16,
 
+    /// Sentry URL.
+    #[config(default_t = None)]
+    pub sentry_url: Option<String>,
+
+    /// Environment name to be used for Sentry.
+    #[config(default_t = None)]
+    pub sentry_environment: Option<String>,
```

Comment:
I don't like the idea of having to manually set the correct environment in every deployment. We have `POD_NAMESPACE` env variable - I think we should use this and default to `unknown/localhost`.

This made me think - how do we set the environent in opsGenie? I assume it's set somewhere from infdra, not from within the app.. Ideally they'd be identical, is it possible?